src/components/Index.js

```diff
@@ -0,0 +1,50 @@
+import { createSuitMixin } from '../mixins/suit';
+import { createWidgetMixin } from '../mixins/widget';
+import indexWidget from 'instantsearch.js/es/widgets/index/index';
+
+// wrapped in a dummy function, since indexWidget doesn't render
+const connectIndex = () => indexWidget;
+
+export default {
+  name: 'AisIndex',
+  mixins: [
+    createSuitMixin({ name: 'Index' }),
+    createWidgetMixin({ connector: connectIndex }),
+  ],
+  provide() {
+    return {
+      // The widget is created & registered by widgetMixin, accessor is needed
+      // because provide is not reactive.
+      $_ais_indexWidget: () => this.widget,
+    };
+  },
+  props: {
+    indexName: {
+      type: String,
+      required: true,
+    },
+    indexId: {
+      type: String,
+      required: false,
+    },
+  },
+  render(createElement) {
+    return createElement(
+      'div',
+      {
+        class: {
```

Comment:
An index doesn't have class names in other flavors. I don't think we should move away from the other flavor implementations by adding more rendering differences.

Reply:
In React InstantSearch it used to have a class. Since we are forced to have something here I don't think it's a big deal to add the class

Reply:
It broadens the API surface in the end. Users mights rely on this class while they don't need it. If we add it now, we'll have trouble removing it later.

We shouldn't add a class just because it's technically not possible to have a render-less widget in Vue. I see having a plain `div` as an implementation detail (since you said it was not possible to not render anything), adding a CSS class makes it turn into an API.